gui-apps/grim/grim-1.2.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Grab images from a Wayland compositor."
+HOMEPAGE="https://github.com/emersion/grim"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/emersion/${PN}.git"
+else
+	SRC_URI="https://github.com/emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="+man jpeg"
+
+DEPEND="
+	>=dev-libs/wayland-protocols-1.14
+	dev-libs/wayland
+	jpeg? ( virtual/jpeg )
```

Comment:
This USE flag has no function in this ebuild, 
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs
(add it as 'optfeature' instead)
((besides I have a feeling it gets pulled by some of the deps, which would make this USE flag even worse))

Reply:
That's because I forgot to propagate it in `emesonargs`, fixed :)